docs/tidb_http_api.md

```diff
@@ -165,12 +165,34 @@
     curl http://{TiDBIP}:10080/schema/{db}
     ```
 
+    ```shell
+    curl http://{TiDBIP}:10080/schema/{db}?id_name_only=true
```

Comment:
`id_name_only=true` seems not RESTFUL...

Reply:
https://stackoverflow.com/questions/39979533/how-should-rest-api-accept-boolean-values here suggested that use `true` to represent boolean arguments

Reply:
I prefer the design like `schema/{db}?fields=id,name`, but it is OK for now since keyvis is the only user.